runtime/src/main/java/org/corfudb/runtime/object/VersionLockedObject.java

```diff
@@ -155,13 +161,20 @@ public VersionLockedObject(Supplier<T> newObjectFn,
      * gc and external object access is needed.
      */
     public void gc(long trimMark) {
+        Timer vloGcTimer = VloMetricsParams.metrics.timer(VloMetricsParams.VLO_GC);
```

Comment:
Could you plaese provide method `VloMetricsParams.getGcTimer()` instead of this

Reply:
done